src/hipscat/pixel_math/healpix_pixel.py

```diff
@@ -98,3 +92,43 @@ def dir(self) -> int:
 
 
 INVALID_PIXEL = HealpixPixel(-1, -1)
+
+
+def get_lower_order_pixel(order: int, pixel: int, delta_order: int) -> int:
```

Comment:
I think this is a pretty common function, and that having this so that we don't need to have a HealpixPixel object to perform the conversion is useful.